src/D2L.Security.OAuth2/Validation/Request/RequestAuthenticator.cs

```diff
@@ -14,14 +15,16 @@ internal sealed partial class RequestAuthenticator : IRequestAuthenticator {
 			m_accessTokenValidator = accessTokenValidator;
 		}
 
-		Task<ID2LPrincipal> IRequestAuthenticator.AuthenticateAsync(
+		[GenerateSync]
+		async Task<ID2LPrincipal> IRequestAuthenticator.AuthenticateAsync(
 			HttpRequestMessage request
 		) {
 			string bearerToken = request.GetBearerTokenValue();
 
-			return AuthenticateAsync( bearerToken );
+			return await AuthenticateAsync( bearerToken ).ConfigureAwait(false);

```

Comment:
I think the latest version of the generator doesn't require this change to `async`, right?